reana-ui/src/components/Footer.js

```diff
@@ -22,6 +23,11 @@ export default function Footer() {
     <footer className={styles["footer-bottom"]}>
       <span>Copyright © 2020 CERN</span>
       <span className={styles["links"]}>
+        {config.cernRopo && (
+          <Link to="/privacy-policy">
+            <Icon name="privacy"></Icon> Privacy Policy
```

Comment:
Hmm, it would be "Privacy policy" with second "p" lowercase (using British English), but since at CERN it is called "Privacy Notice", let's perhaps change to "Privacy Notice"?